.github/workflows/publish.yml

```diff
@@ -0,0 +1,12 @@
+name: Publish to pub.dev
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+jobs:
+  publish:
+    uses: dart-lang/setup-dart/.github/workflows/publish.yml@v1
+    # with:
+    #   working-directory: path/to/package/within/repository
```

Comment:
i'm not sure these are meant to be comments but that's what was in the pub article snippet and I can't find an example anywhere.